varats/projects/cpp_projects/doxygen.py

```diff
@@ -16,20 +16,22 @@
 
 from varats.paper.paper_config import project_filter_generator
 from varats.utils.project_util import wrap_paths_to_binaries, block_revisions, \
-    BlockedRevisionRange
+    BugAndFixPair
```

Comment:
```suggestion
    BugAndFixPair)
```
needs to be applied with the previous suggestion